packages/design-system/src/atoms/tooltipV2/index.tsx

```diff
@@ -4,7 +4,7 @@ import { TypographyV2 } from '../typographyV2'
 
 export interface TooltipV2Props
   extends Omit<TooltipProps, 'title' | 'variant'> {
-  title: string
+  title: string | undefined
```

Comment:
That's not correct. Tooltips should always have a title, otherwise, it'll render an empty container. Instead, you have to check in the ButtonV2 component & show the tooltip only if the `tooltip` prop is defined.